client/db/types.go

```diff
@@ -71,17 +74,26 @@ type AccountInfo struct {
 	FeeCoin   []byte
 	// Paid will be set on retrieval based on whether there is an AccountProof
 	// set.
-	Paid bool
+	Paid         bool
+	ClientPubKey *secp256k1.PublicKey
```

Comment:
`ClientPubKey` looks to be unused.  But the private key (and thus the public key and AccountID) are retrievable via `EncKey`.

Reply:
Thanks, I'll clean that up.